build_scripts/rollback_canary_tier_1.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+readonly DISTRIBUTION_ID="E3CL6P6NHSXGR6";
+readonly MIN_FILE="keen-tracking.min.js";
+readonly S3_BUCKET="wespire-tracking-library";
+
+for arg in "$@"
+do
+    if [ "$arg" == "--force" ] || [ "$arg" == "-f" ]
+    then
+        FORCE_DEPLOY="yes"
+        echo "Using --force option (non-interactive mode)."
+    fi
+done
+
+if [ "$FORCE_DEPLOY" != "yes" ]
+then
+  while true; do
+      read -p "About to rollback build to previous version. Confirm? (Yes/No) " yn
+      case $yn in
+          [Yy]* ) break;;
+          [Nn]* ) echo "Ok, exiting."; exit;;
+          * ) echo "Please answer yes or no.";;
+      esac
+  done
+fi
+
+if aws --version &> /dev/null
+then
+  echo "rolling back canary";
+  aws s3 cp "s3://${S3_BUCKET}/keen-tracking_last.min.js" "s3://${S3_BUCKET}/${MIN_FILE}" --region us-east-2 --acl public-read
```

Comment:
note: in the other file, variable `BACKUP_FILE` was used and in this file it's hardcoded.